storage/kv.go

```diff
@@ -0,0 +1,116 @@
+package storage
+
+import (
+	"encoding/binary"
+	"log"
+	"time"
+
+	"github.com/coreos/etcd/storage/backend"
+	"github.com/coreos/etcd/storage/storagepb"
+)
+
+var (
+	batchLimit    = 10000
+	batchInterval = 100 * time.Millisecond
+	keyBucketName = []byte("key")
+)
+
+type store struct {
+	b       backend.Backend
+	kvindex index
+
+	now        uint64 // current index of the store
```

Comment:
can we call this something more meaningful? currentIndex? horizon? "now" seems like a timestamp